src/v/config/configuration.cc

```diff
@@ -3174,7 +3174,15 @@ configuration::configuration()
       "Minimum number of active producers per virtual cluster",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       std::numeric_limits<uint64_t>::max(),
-      {.min = 1}) {}
+      {.min = 1})
+  , enable_consumer_offsets_delete_retention(
```

Comment:
nit: may want to consider prefixing this with "unsafe" or something. The resulting behavior for group management seems undefined in the general case, without additional state management